dev-python/backports-shutil_which/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+		<email>python@gentoo.org</email>
```

Comment:
I thought this might happen. I only included the metadata.xml file because repoman complained about it. I could probably proxy maintain it with some guidance.

Thanks for the review.